Sources/Public/EdgeBridge+PublicAPI.swift

```diff
@@ -0,0 +1,48 @@
+//
+// Copyright 2022 Adobe. All rights reserved.
+// This file is licensed to you under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License. You may obtain a copy
+// of the License at http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under
+// the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+// OF ANY KIND, either express or implied. See the License for the specific language
+// governing permissions and limitations under the License.
+//
+
+import AEPCore
+import AEPServices
+import Foundation
+
+/// Defines the public interface for the EdgeBridge extension
+public extension EdgeBridge {
+    /// Starts context data capture session, hooking into Edge Bridge's dispatch of events.
+    public static func startContextDataCaptureSession() {
+        // Dispatch event to edgebridge to signal start capture of context data
+        let event = Event(name: EdgeBridgeConstants.EventNames.EDGE_BRDIGE_START_CONTEXT_DATA_CAPTURE,
+                          type: EventType.edgeBridge,
+                          source: EventSource.startCapture,
+                          data: nil)
+
+        MobileCore.dispatch(event: event)
+    }
+    
+    /// Stops context data capture session, outputting the merge result using the case sentivity setting applied, and removing captured events from memory.
```

Comment:
Maybe change to:

> Stops a debug context data capture session and prints a report of the captured context data keys and values during the session to the device logs. The report can then be uploaded to Data Prep for Data Collection to map the context data to XDM.
> Note, this API is intended for use during debugging and should not be used in a production environment. The SDK log level must be configured to LogLevel.debug or LogLevel.trace for the report to be printed.

Reply:
Do you guys think adding an option to "writeToFile" be helpful for the user?

Reply:
We can take that as an enhancement task after we close this.